package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "agoric-basics",
+  "version": "0.1.0",
+  "license": "Apache-2.0",
+  "private": true,
+  "useWorkspaces": true,
+  "workspaces": [
+    "contract"
+  ],
```

Comment:
Hm. no endo resolutions to work around https://github.com/Agoric/agoric-sdk/issues/8621

I was expecting something started with `yarn create @agoric/dapp --dapp-template dapp-offer-up` with support for starting the chain in docker, deploying the contract, testing that it's not >1MB compressed, etc.

Reply:
added an issue:

 - https://github.com/Agoric/dapp-agoric-basics/issues/4